.depcheckrc

```diff
@@ -9,4 +9,5 @@ ignores: [
     "eslint-*",
     "jest",
     "size-limit",
+    "json2csv",
```

Comment:
i think there's also a way to configure ignoring of folders, so that stories related ones don't trigger any of the issues. minor though

Reply:
> --ignore-dirs: DEPRECATED, use ignore-patterns instead. A comma separated array containing directory names to ignore. Example, --ignore-dirs=dist,coverage.

> --ignore-patterns: Comma separated patterns describing files to ignore. Patterns must match the .gitignore spec. Example, --ignore-patterns=build/Release,dist,coverage,*.log

doesn't seem like a good idea seeing that it should match `.gitignore` so I didn't include it in this one